src/spider/worker/task_executor.cpp

```diff
@@ -112,6 +116,18 @@ auto main(int const argc, char** argv) -> int {
         return cCmdArgParseErr;
     }
 
+    // Ignore SIGTERM
```

Comment:
Although we did say letting the task finish made sense, let's double check the ideal behaviour with @jackluo923 before we finish.